### DISCUSSION TITLE
Regarding to MongoDB Blogs

### DISCUSSION THREAD

**Discussion by one-click-varshil-doshi** [dailydotdev/daily#516] [General]
Hi daily.dev,
I am a software developer(front-end & back-end). I just want know that how to use mongoDB with any front-end framework like angular as a database technology.If daily.dev can arrange some kind of document or blog then it would be Great.
Thanks daily.dev

**Comment by idoshamun** [comment]
Hi @one-click-varshil-doshi,

We don't create the content ourselves but feel free to check out the MongoDB tag with all the associated articles.
https://app.daily.dev/tags/mongodb
